src/main/java/com/conveyal/r5/streets/StreetLayer.java

```diff
@@ -94,6 +94,8 @@
      */
     public static final double LINK_RADIUS_METERS = 300;
```

Comment:
Consider increasing?

https://github.com/conveyal/r5/issues/377

Reply:
Agreed, as mentioned in #377 we should just use one symbolic constant for linking distance on both ends of routing. We've decided to make this change later though so we can get a release out, and produce multiple worker releases with different characteristics in case a client needs them.